index.bs

```diff
@@ -1949,19 +1949,21 @@ Methods</h4>
 		event named <code>complete</code> for legacy reasons.
 
 		<div algorithm="OfflineAudioContext.startRendering()">
+			Let <dfn attribute for="OfflineAudioContext">[[rendering started]]</dfn> be an internal slot of this {{OfflineAudioContext}}.  Initialize this slot to <em>false</em>.
```

Comment:
Don't know why, but `[[renderingStarted]]` doesn't get processed correctly by bikeshed.  It thinks the slot is called `[renderingStarted]`,  Changing it to `[[rendering started]]` makes everything look right.